perception/autoware_compare_map_segmentation/CMakeLists.txt

```diff
@@ -22,11 +22,11 @@ include_directories(
 )
 
 add_library(${PROJECT_NAME} SHARED
-  src/distance_based_compare_map_filter/node.cpp
-  src/voxel_based_approximate_compare_map_filter/node.cpp
+  # src/distance_based_compare_map_filter/node.cpp
+  # src/voxel_based_approximate_compare_map_filter/node.cpp
   src/voxel_based_compare_map_filter/node.cpp
-  src/voxel_distance_based_compare_map_filter/node.cpp
-  src/compare_elevation_map_filter/node.cpp
+  # src/voxel_distance_based_compare_map_filter/node.cpp
+  # src/compare_elevation_map_filter/node.cpp
```

Comment:
This seems like a very temporary workaround. I think it would be better to make it unnecessary to comment out the above before merging. What do you think?

Reply:
Or do you mean that the commented-out feature is intended to be deprecated?

Reply:
@yukkysaito I'm sorry, it was my typo. I fixed https://github.com/autowarefoundation/autoware_universe/pull/10299/commits/89523c90e8b95a7334d1a9cd533c7f9627026dfe
By the way, we also started a discussion about deprecation of unused filters. I will open the disucssion later.